client/src/components/ratingsReviews/RatingsAndReviews.jsx

```diff
@@ -109,8 +107,8 @@ class RatingsAndReviews extends React.Component {
   }
 
   getDefaultRatings() {
-    const { ratings, characteristics, recommended, averageRating, totalRating } = this;
-    this.props.updateRatings(ratings, characteristics, recommended, averageRating, totalRating);
+    const { ratingsData, ratings, characteristics, recommended} = this;
+    this.props.updateRatings(ratingsData, ratings, characteristics, recommended);
```

Comment:
Same thing. Don't worry about splitting up this data.

Reply:
Done